addons/l10n_pe_website_sale/static/src/js/website_sale.js

```diff
@@ -0,0 +1,73 @@
+/** @odoo-module **/
+
+import { WebsiteSale } from "@website_sale/js/website_sale";
+
+WebsiteSale.include({
+    events: Object.assign({}, WebsiteSale.prototype.events, {
+        "change select[name='state_id']": "_onChangeState",
+        "change select[name='city_id']": "_onChangeCity",
+    }),
```

Comment:
Hello @JulienVR thanks for this feedback, yeah, I guess I didn't see this one since I used a different State/Province and using another one works correctly. @desdelinux told me they're working on the fixes, as soon as they're done, I'll test again and let you know my findings. Thanks for the review!

Cheers!

Fernando

Reply:
Hello @JulienVR and @fvz-odoo 

I've been working on this particular change and the way Odoo works natively made it a bit difficult since the functions `_onChangeCountry` and `_changeCountry` don't return anything. Which is why a made [these](https://github.com/odoo/odoo/pull/130155/commits/0e47e7085bcb3168c48555a1eaccbc8234078184#diff-8545ff07230315c44a6ad5a96a900b2d9e6c8f104be5e19d60e2ececd0ed5190R246) changes. And with that I can simply do the [following](https://github.com/odoo/odoo/pull/130155/commits/d10206f2e367d5d8af24f12ff4c0b6be44dfe278#diff-3e73cdaf8077b89c48579b44b997cff31cf049b58fc66ed417209e307f5b0de4R83).

What do you guys think about these changes? Are they a valid improvement to make these functions inheritable? Or should I leave it as it is and override and repeat some code?